python/ray/serve/_private/replica.py

```diff
@@ -321,6 +321,9 @@ def _configure_logger_and_profilers(
             component_id=self._component_id,
         )
 
+    def push_proxy_handle(self, handle):
```

Comment:
Should we do something to the handle? Also maybe add a type hint is it's required 🙃

Reply:
doing something with the handle seems unnecessary for now, I think if you pass any actor handle as an argument in a ray remote call like:
```
x.remote(actor_handle)
```
then ray core does some processing under the hood that requires making a call to the GCS, so if this `actor_handle` was never "pushed" to actor beforehand then this call hangs. "Pushing" it once is enough to unblock the call though when the GCS goes down.